### ISSUE TITLE
Chef::Exceptions::FileNotFound when using "mongodb_instance" LWRP in recipe

### ISSUE THREAD

**Issue by toxsick** [edelight/chef-mongodb#106]
Hello,

i try to use the "mongodb_instance" LWRP from within a recipe in my own cookbook. My Myapp::database recipe looks like this:

``` ruby
# add 10-gen repo
cookbook_file "/etc/yum.repos.d/10gen.repo" do
  source "10gen.repo"
  mode 0644
  owner "root"
end

# install mongodb
include_recipe "mongodb::default"

directory "/fuf/pxstdb" do
  owner "mongod"
  group "mongod"
  mode 0700
end

mongodb_instance "pxstdb" do
  dbpath "/fuf/pxstdb"
  cookbook_name "mongodb"
end
```

However it seem that mongodb_instance tries to find the template in my cookbook and not the mongodb cookbook:

https://gist.github.com/hansotronic/5564421

Any thoughts on that??

**Comment by josephholsten** [comment]
Sounds like you're having the same issue as https://github.com/edelight/chef-mongodb/pull/103, which totally seems like a bug to me.

**Comment by jamesonjlee** [comment]
we merged in #103 does this still happen?

**Comment by jamesonjlee** [comment]
asumming no,
